docs/en/your-profile-and-preferences/customize-group-chat-notifications.md

```diff
@@ -8,9 +8,65 @@ hide:
 
 # Customize group chat notifications
 
- :octicons-tools-24: In progress
+You can customize the messages and mentions notifications you receive in group chats. By doing so, you only receive the group chat notifications relevant to you.
 
-!!! note ""
-     We're working on this content.
+## Mute notifications in specific group chats
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-messages: **Messages**.
+    1. Long press the group chat you want to mute.
+    1. Tap :mobile-mute: **Mute**.
+    1. Select the length of time you'd like to mute your notifications.
+
+=== "Desktop"  
+
+    1. From the navigation sidebar, click :desktop-messages: **Messages**.
+    1. Under the messages list , right click the group you want to mute.
+    1. Select **Mute chat**.
+
+!!! tip
+	  Group chats show up greyed out :mobile-mute: **Mute** next to their name, when you mute notifications.
+
+## Mute group chat notifications except personal mentions
+
+If you want to avoid missing important messages, you can selectively mute some notifications, except global mentions. This procedure mutes all group chats notifications except personal mentions
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-communities: **Messages**.
+    1. Long press the group chat you want to mute.
+    1. Tap :mobile-notifications-settings: **Notification settings**.
+    1. Select **Global mentions** > **Notifications off**.
+    1. Select **All other messages** > **Notifications off**.
+    1. Verify **Personal mentions** is set to **On**. If it is not, then select **Personal mentions** > **Notifications on**.
+
+=== "Desktop"  
+
+    1. From the navigation sidebar, click :desktop-settings: **Settings**.
+    1. From the **Settings sidebar**, click :desktop-notification: **Notifications and Sounds**.
+    1. Scroll down to  the **Exceptions** area and click :desktop-add: **Add** next to the group chat you want to mute.
+    1. Select **Global mentions** > **Turn off**. 
+    1. Select **All other messages** > **Turn off**.
+    1. On the same pop-up menu, verify **Personal mentions** is set to **Send alerts**. If it is not, then select **Personal mentions** > **Send Alerts**.
```

Comment:
- 'On the same...' -> 'In the same'
- 'verify' -> 'make sure'
- 'If it is not' -> 'If it's not'